src/praisonai/praisonai/gateway/kanban_dispatcher.py

```
@@ -0,0 +1,417 @@
+            # Build command to execute task
+            # This could be configurable, but for now use a simple approach
+            cmd = self._build_execution_command(task)
+            
+            logger.info(f"Spawning worker for task {task.id}: {' '.join(cmd)}")
+            
+            # Start process with output redirect to avoid deadlock
+            import tempfile
+            with tempfile.NamedTemporaryFile(mode='w+', delete=False, suffix='.log') as temp_log:
+                temp_log_path = temp_log.name
+            
+            process = subprocess.Popen(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)